### PR TITLE
xsd changes

### PR DESCRIPTION
**Pull request by norrismiv**
loot, territory, roles, npcs, etc

### CODE REVIEW

File: HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd [hunk 1]
```diff
@@ -158,14 +158,7 @@
...
   <xs:complexType name="LootImport">
-    <xs:simpleContent>
-      <xs:extension base="hyb:String8">
-        <xs:attribute name="Rolls" type="xs:int" default="1" use="optional"/>
-        <xs:attribute name="Chance" type="xs:float" default="1" use="optional"/>
-        <xs:attribute name="Override" type="xs:boolean" default="false" use="optional"/>
-        <xs:attribute name="Guaranteed" type="xs:boolean" default="false" use="optional"/>
-      </xs:extension>
-    </xs:simpleContent>
+    <xs:attribute name="Name" type="hyb:String8"/>
```
Comment (baughj) [inline HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd]: I think import can go completely away now:

File: HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd [hunk 2]
```diff
@@ -216,8 +206,103 @@
...
+    </xs:sequence>
+    <xs:attribute name="Receive" type="hyb:String8"/>
+  </xs:complexType>
+
+  <xs:complexType name="NpcRoleRepair">
+    <xs:attribute name="Nation" type="hyb:String8"/>
+    <xs:attribute name="Discount" type="xs:int"/>
+    <xs:attribute name="Type">
+      <xs:simpleType>
+        <xs:list itemType="hyb:String8"/>
```
Comment (baughj) [inline HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd]: This should probably be an enum?

File: HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd [hunk 3]
```diff
@@ -216,8 +206,103 @@
...
+      <xs:element name="Post" type="creature:NpcRolePost" maxOccurs="1" minOccurs="0" />
+      <xs:element name="Repair" type="creature:NpcRoleRepair" maxOccurs="1" minOccurs="0" />
+      <xs:element name="Bank" type="creature:NpcRoleRepair" maxOccurs="1" minOccurs="0" />
+    </xs:sequence>
+  </xs:complexType>
+
+  <xs:complexType name="Npc">
+    <xs:sequence>
+      <xs:element name="Name" minOccurs="1" maxOccurs="1" type="hyb:String8" />
+      <xs:element name="Displayname" minOccurs="0" maxOccurs="1" type="hyb:String8" />
```
Comment (baughj) [inline HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd]: Following convention elsewhere this should be `DisplayName`

File: HybrasylIntegration/HybrasylXML/XSD/Loot.xsd [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+<xs:schema xmlns:hyb="http://www.hybrasyl.com/XML/HybrasylCommon" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:loot="http://www.hybrasyl.com/XML/Loot" targetNamespace="http://www.hybrasyl.com/XML/Loot" elementFormDefault="qualified">
+
+  <xs:annotation>
+    <xs:documentation xml:lang="en">
+      Hybrasyl Project - Hybrasyl XML Schema - Loot Types
+      This file is subject to the same licenses as Project Hybrasyl.
+      (C) 2016 Project Hybrasyl (info@hybrasyl.com)
+      Written by Michael Norris (norrism@hybrasyl.com)
+    </xs:documentation>
+  </xs:annotation>
```
Comment (baughj) [inline HybrasylIntegration/HybrasylXML/XSD/Loot.xsd]: awesome, the header is right :p

File: HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd (hunk 3 above)
Comment (norrismiv) [inline HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd]: Name in this case is referring to the name key for the created object. Not the display name.

File: HybrasylIntegration/HybrasylXML/XSD/Loot.xsd (hunk 4 above)
Comment (norrismiv) [inline HybrasylIntegration/HybrasylXML/XSD/Loot.xsd]: I du gud.

File: HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd (hunk 2 above)
Comment (norrismiv) [inline HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd]: These are the actual Role elements i.e. Train, Vend, Repair, etc

File: HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd (hunk 1 above)
Comment (norrismiv) [inline HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd]: Import has gone away, Set has replaced it. LootImport object remains, in xml it becomes `<Set name=""`

Comment (baughj) [inline HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd]: OK.

File: HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd (hunk 2 above)
Comment (baughj) [inline HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd]: This is a type attribute with a list of strings in the repair tag; that's what I'm referring to.

File: HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd (hunk 3 above)
Comment (baughj) [inline HybrasylIntegration/HybrasylXML/XSD/Creatures.xsd]: Again, referring to `Displayname` versus `DisplayName`

**Pull request merged**